common-design-patterns/breadcrumb-trail.md

```diff
@@ -0,0 +1,242 @@
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/header/header-docs-en.html">GCWeb (WET) header documentation</a></li>
+  </ul>
+  <h3>Implementations</h3>
+  <p>Determine what best suits the type of page you're creating. Refer to your implementation's guidance if you want to exclude breadcrumbs.</p>
+  <div class="row">
+    <div class="col-md-8">
+      <div class="wb-tabs mrgn-tp-lg">
+        <div class="tabpanels">
+          <details id="004" open="open">
+            <summary><strong>GC-AEM</strong></summary>
+            <p class="mrgn-tp-lg">For the Government of Canada Adobe Experience Manager (AEM):</p>
+            <ul>
+              <li><a href="https://www.gcpedia.gc.ca/gcwiki/images/9/9a/AEM-6.5-Documentation-Unit-3-7-Changing-the-Default-Breadcrumb.pdf">Changing the default breadcrumb (PDF - GCPedia link - only available on the Government of Canada network)</a></li>
+              <li><a href="https://www.gcpedia.gc.ca/wiki/AEM_GC-specific_Documentation_6.5">AEM/Managed Web Service documentation (GCPedia link - only available on the Government of Canada network)</a></li>
+            </ul>
+          </details>
+          <details id="005">
+            <summary><strong>CDTS</strong></summary>
+            <p class="mrgn-tp-lg">For the Centrally Deployed Templates Solution (CDTS):</p>
+            <ul>
+              <li><a href="https://cdts.service.canada.ca/app/cls/WET/gcweb/v4_0_47/cdts/samples/breadcrumbs-en.html">Breadcrumbs - CDTS documentation </a></li>
+              <li><a href="https://cenw-wscoe.github.io/sgdc-cdts/docs/index-en.html">CDTS documentation</a></li>
+            </ul>
+          </details>
+          <details id="006">
+            <summary><strong>Drupal WxT</strong></summary>
+            <p class="mrgn-tp-lg">For Drupal WxT:</p>
+            <ul>
+              <li><a href="https://drupalwxt.github.io/en/">Drupal WxT documentation</a></li>
+            </ul>
+          </details>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="cnt-wdth-lmtd">
+    <h2 id="research">Research and rationale</h2>
+    <p>Consult research findings and policy rationale.</p>
+    <h3>Research findings</h3>
+    <p><a href="https://blog.canada.ca/2020/08/10/CanadaDotCa-trusted-source.html">Canada.ca is a trusted source</a><br>
+      Explains the decision to use “Canada.ca” as the label for the first link in the breadcrumb.</p>
+    <p><a href="https://blog.canada.ca/research-summaries/wayfinding-on-canada-ca.html">Wayfinding on Canada.ca research summary</a><br>
+      Research shows that people navigating on the site use breadcrumb links nearly twice as often as they use the Theme and topic menu.</p>
+    <p>The spacing specifications for the breadcrumb links are designed so that touch targets meet WCAG AAA requirements.</p>
+    <h3>Policy rationale</h3>
+    <p>As part of the global header, the breadcrumb is a mandatory element under the Content and Information Architecture Specification.</p>
```

Comment:
```suggestion
        <p>The spacing specifications for the breadcrumb links are designed so that touch targets meet WCAG AAA requirements.</p>
        <p>As part of the global header, the breadcrumb is a mandatory element under the Content and Information Architecture Specification.</p>
```

Reply:
Inserted paragraph re WCAG req't

Reply:
Done